plugins/plugins64.xml

```diff
@@ -341,6 +341,30 @@
 			</x64>
 		</install>
 	</plugin>
+	<plugin name="SQLinForm SQL Formatter">
+                <x64Version>5.6.63</x64Version>
+                <aliases>
+                     <alias name="SQLinForm"/>
+                </aliases>
+                <description>SQLinForm formats plain SQL, SQL embedded in Program Code, SQL snippets and SQL Statements with syntax errors.\n SQLinForm's Plugin for Notepad++ has a powerful code folding feature and formats your SQL as you Type.
+                             It supports all major DB like Oracle, DB2, SQL Server, Teradata, Netezza, MySQL, Sybase, MS Access, etc. SQLinForm incorporates its own multi-DB parser engine. No additional software is required.</description>
+                <author>Guido Thelen</author>
+                <homepage>http://www.sqlinform.com</homepage>
+                <latestUpdate>Added 64bit support</latestUpdate>
+                <stability>Good</stability>
+                <install>
+                        <x64>
+                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.zip</download>
```

Comment:
@guidomarcel  Could you also provide the 32bit version as zip, so I could add it also to the list for 32bit plugins at https://npppm.bruderste.in/plugins